examples/try_everything.py

```diff
@@ -52,53 +55,69 @@ async def test_file_stream(request):
 
 @app.exception(ServerError)
 async def test(request, exception):
-    return response.json({"exception": "{}".format(exception), "status": exception.status_code},
-                         status=exception.status_code)
+    return response.json(
+        {"exception": "{}".format(exception), "status": exception.status_code},
```

Comment:
Could use `dict(exception=str(exception), status=exception.status_code)`, or at least avoiding the `format`

Reply:
I did not even look at this.

I think this is probably the easiest to read:
```python
{"exception": str(exception), "status": exception.status_code},
```

Reply:
I agree on that.